docs/source/scale/sagemaker.rst

```diff
@@ -54,9 +54,9 @@ The ``build_docker_sagemaker.sh`` script takes four arguments:
 .. warning::
     In order to upload the GraphStorm SageMaker Docker image to Amazon ECR, users need to define the <DOCKER_NAME> to include the ECR URI string, **<AWS_ACCOUNT_ID>.dkr.ecr.<REGION>.amazonaws.com/**, e.g., ``888888888888.dkr.ecr.us-east-1.amazonaws.com/graphstorm``.
 
-4. **DOCKER_TAG** (optional), is the assigned tag name of the to-be built Docker image. Default is ``sm``.
+4. **DOCKER_TAG** (optional), is the assigned tag name of the to-be built Docker image. Default is ``sm-<cpu/gpu>``.
```

Comment:
`sm-<cpu/gpu>` is a little bit confusing. Let's use `sm-cpu` and `sm-gpu` for cpu image and gpu image respectively.